docs/DEVELOPING.md

```diff
@@ -343,6 +343,144 @@ $ make ftest NAME=azure_blob_storage
 
 ℹ️ The e2e test uses default values defined in [configure Azure Blob Storage connector](#configure-azure-blob-storage-connector)
 
+
+## PostgreSQL Connector
+
+The [Elastic PostgreSQL connector](../connectors/sources/postgresql.py) is provided in the Elastic connectors python framework and can be used via [build a connector](https://www.elastic.co/guide/en/enterprise-search/current/build-connector.html).
+
+### Availability and prerequisites
+
+⚠️ _Currently, this connector is available in **beta** starting in 8.7_.
+Features in beta are subject to change and are not covered by the service level agreement (SLA) of features that have reached general availability (GA).
+
+Elastic versions 8.6.0+ are compatible with Elastic connector frameworks. Your deployment must include the Elasticsearch, Kibana, and Enterprise Search services.
+
+PostgreSQL versions 11 to 15 are compatible with Elastic connector frameworks.
+
+**Prerequisite**
+
+- PostgreSQL user should hold the ownership of the tables to indexed the data.
+- PostgreSQL user needs superuser privilges to index all the tables of the database.
+- User needs to set `track_commit_timestamp` to `on` using `ALTER SYSTEM SET track_commit_timestamp = on;` query to fetch last updated time.
```

Comment:
Can be clearer and more explicit about where the user sets these configurations.

Reply:
Updated the sentence.